exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServer.java

```diff
@@ -157,7 +165,7 @@ InetSocketAddress getAddress() {
     return server.getAddress();
   }
 
-  private static class MetricsHandler implements HttpHandler {
```

Comment:
I'm debating in my head about whether I prefer to inject the mutated field or to make this non-static.  How feasible is it for the handler to own the set of duplicated names, rather than having it mutated like this?

Reply:
This was a silly mistake of mine. The field can be moved to be a member of `MetricHandler` since its unused by `PrometheusHttpServer`, and `MetricHandler` can be reverted to static.